tools/build_config/Jenkinsfile

```diff
@@ -304,6 +316,11 @@ pipeline {
         ]
       )
 
+      archiveArtifacts(
```

Comment:
can this be merged back into the other archive call, or is there clarity in keeping it separate?

Reply:
I think there is clarity in separation as they're done on different stages.